lib/project_types/script/messages/messages.rb

```diff
@@ -50,6 +50,25 @@ module Messages
 
           script_not_found_cause: "Couldn't find script %s for extension point %s",
 
+          service_failure_cause: "Internal service error. " \
+                                 "Ensure the 'shopify/scripts-toolchain-as' package is up to date.",
+
+          metadata_validation_cause: "Invalid script extension metadata. " \
+                                     "Ensure the 'shopify/scripts-toolchain-as' package is up to date and " \
+                                     "'package.json' contains a 'scripts/build' entry with a " \
+                                     "'--metadata build/metadata.json' argument",
+
+          metadata_schema_versions_missing: "Invalid script metadata:" \
+                                            " 'schemaVersions' field is missing",
+          metadata_schema_versions_single_key: "Invalid script extension metadata:" \
+                                               " 'schemaVersions' can have only one extension point name.",
+          metadata_schema_versions_missing_major: "Invalid script extension metadata:" \
+                                                  " 'schemaVersions' is missing the 'major' field",
+          metadata_schema_versions_missing_minor: "Invalid script extension metadata:" \
+                                                  " 'schemaVersions' is missing the 'minor' field",
+
+          metadata_not_found_cause: "Script version file (%s) cannot be found.",
```

Comment:
The help suggestion for this error should be the one that's currently specified for the metadata validation error (i.e., toolchain being up to date and the metadata flag being present in the build command).